sdk/libs-scala/contextualized-logging/BUILD.bazel

```diff
@@ -15,7 +15,7 @@ da_scala_library(
         "@maven//:org_apache_pekko_pekko_stream",
         "@maven//:io_spray_spray_json",
     ],
-    scalacopts = ["-Xsource:2.13"],
+    scalacopts = ["-Xsource:3"],
```

Comment:
Why do we need this one, this is not part of the drop ?

Reply:
Left overs from prior resolution attempts - I'll revert this change and the other 2. Thanks for pointing out.